lfucg-exactions/server/plats/utils.py

```diff
@@ -0,0 +1,10 @@
+def created_by_modified_by(self, **kwargs):
```

Comment:
Look into the perform_save and perform_update hooks instead of overriding each serializer.